### PR TITLE
Update Policy

### PR DESCRIPTION
**Pull request by pickjasmine**
Dependent on some work that @aalsabag is doing in the branch `updatePolicy` in rode

I'm making the assumption that updating a policy with invalid rego code will be handled the same in rode as when you are creating a policy with invalid rego, but I have not had a chance to manually test that yet so that path may not be complete yet.

### CODE REVIEW

File: pages/policies/[id].js [hunk 1]
```diff
@@ -17,35 +17,45 @@
...
       <div className={styles.detailsContainer}>
         <Loading loading={loading}>
-          {data ? (
+          {policy ? (
             <>
-              <div>
-                <p className={styles.policyName}>{data.name}</p>
-                <p className={styles.policyDescription}>{data.description}</p>
+              <div className={styles.detailsHeader}>
+                <div>
```
Comment (aalsabag) [inline pages/policies/[id].js:47]: Can we just define this div to extend to the Edit Policy button? Or whatever solution is recommended.

What's happening now is that the policyName dictates the size of the div. The subsequent policyDescription has a max-width of 90%. That's 90% of the div defined by the policyName. So a small policyName means the description can be many lines long

File: pages/policies.js [hunk 2]
```diff
@@ -37,6 +37,8 @@ const createSearchFilter = (query) => {
   return null;
 };
 
+// TODO: handle flashing of not found on this and resource search page
```
Comment (aalsabag) [inline pages/policies.js:40]: Shouldn't be an issue any more after the next PR

**Review by aalsabag — APPROVED** [review]
lgtm. real minor comment

File: pages/policies/[id].js (hunk 1 above)
Comment (pickjasmine) [reply pages/policies/[id].js:47]: Could you post a screenshot of what you're seeing? I have a small policy name with a long description, but it does not look like the constraint is to the policy name like you're saying so I just want to make sure I'm doing you justice

image

Comment (pickjasmine) [reply pages/policies/[id].js]: @aalsabag check out the latest changes I pushed, they might fix the issue you were seeing? I was definitely not handling super long policy names, and your comment reminded me so thanks! Hopefully all should be good now.

**Pull request merged**